tests/src/test/resources/ignored-links

```diff
@@ -27,4 +27,5 @@ http://127.0.0.1:3000/oauth/callback
 https://api.linkedin.com/v2/me
 https://raw.githubusercontent.com/jboss-container-images/redhat-sso-7-openshift-image/sso-cd-dev/templates/${resource}
 https://raw.githubusercontent.com/jboss-container-images/redhat-sso-7-openshift-image/sso73-dev/templates/${resource}
-https://issues.jboss.org/secure/CreateIssueDetails*
\ No newline at end of file
+https://issues.jboss.org/secure/CreateIssueDetails*
+https://expressjs.com/en/guide/behind-proxies.html
```

Comment:
Why are you ignoring the link? This means we won't know if it breaks.

Reply:
@stianst it is breaking at the moment, even though, the link works it returns HTTP 403 on tests. So I assumed that was our way to solve this.